core/sequencer/server.go

```diff
@@ -317,7 +317,7 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 	}
 
 	// Set new leaf values.
-	mapRoot, err := mapClient.SetLeaves(ctx, newLeaves, metadata)
+	mapRoot, err := mapClient.SetLeavesAtRevision(ctx, in.Revision, newLeaves, metadata)
```

Comment:
`err` can be either "such revision exists", or other sort of error (possibly critical). I guess, there is no distinction between them on the call site? Does the call site `Exitf` anywhere up the stack, or skips this error?

Reply:
We don't `Exitf` anywhere.  This run of `RunBatch` will return with the error and the whole thing will be retried.